src/Core/src/Primitives/HybridWebViewInvokeJavaScriptRequest.cs

```diff
@@ -3,11 +3,11 @@
 
 namespace Microsoft.Maui
 {
-	public class HybridWebViewInvokeJavaScriptRequest(string methodName, JsonTypeInfo returnTypeJsonTypeInfo, object?[]? paramValues, JsonTypeInfo?[]? paramJsonTypeInfos)
+	public class HybridWebViewInvokeJavaScriptRequest(string methodName, JsonTypeInfo? returnTypeJsonTypeInfo, object?[]? paramValues, JsonTypeInfo?[]? paramJsonTypeInfos)
```

Comment:
This is technically not breaking, so we can do this for .NET 9. Since this change is attribute-only, we could just secretly allow nulls and make this change later. But, since it is not breaking and the number of people creating their own handler for the hybrid web view is probably 0, we can do this and just make everything easier in the future.